pkg/backup/item_backupper.go

```diff
@@ -435,10 +435,9 @@ func (ib *defaultItemBackupper) takePVSnapshot(obj runtime.Unstructured, log log
 
 	log = log.WithField("volumeID", volumeID)
 
-	tags := map[string]string{
-		"velero.io/backup": ib.backupRequest.Name,
-		"velero.io/pv":     pv.Name,
-	}
+	tags := ib.backupRequest.GetLabels()
+	tags["velero.io/backup"] = ib.backupRequest.Name
```

Comment:
looks like we need to make sure `tags` is not nil after getting the backup's labels before inserting